crypto/x509/x509_lu.c

```diff
@@ -405,6 +405,13 @@ X509_CRL *X509_OBJECT_get0_X509_CRL(X509_OBJECT *a)
     return a->data.crl;
 }
 
+char *X509_OBJECT_get0_data(const X509_OBJECT *a)
+{
+    if (a == NULL)
```

Comment:
This is not necessary since you are not supposed to call the function with NULL.

Reply:
fixed: NULL check is removed